core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -246,11 +248,11 @@ public PlayerInformation getPlayerInformation() {
     @Override
     public void attach(PlayerView playerView) {
         mediaPlayer.setSurfaceHolderRequester(playerView.getSurfaceHolderRequester());
-        BuggyVideoDriverPreventer.newInstance(playerView.getContainerView(), this).preventVideoDriverBug();
+        buggyVideoDriverPreventer.preventVideoDriverBug(this, playerView.getContainerView());
```

Comment:
Inverted the dependency so that we can pass the `Preventer` through the constructor for `mocking`.